workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/Tests/VisualElementConcealerTests.cs

```diff
@@ -0,0 +1,62 @@
+using NUnit.Framework;
+using UnityEngine.UIElements;
+
+namespace Improbable.Gdk.Debug.WorkerInspector.Codegen.EditmodeTests
+{
+    [TestFixture]
+    public class VisualElementConcealerTests
+    {
+        [TestCase(false)]
+        [TestCase(true)]
+        public void HideCollectionEventSetsHideProperty(bool value)
+        {
+            var vis = new VisualElement();
+            var stub = new VisualElementConcealer(vis);
+            stub.HandleSettingChange(new HideCollectionEvent().WithValue(value).WithTarget(vis));
+            Assert.AreEqual(value, stub.HideIfEmpty);
+        }
+
+        [Test]
+        public void ConcealerHidesElementIfHideSettingAndIsHiddenIsTrue()
+        {
+            var vis = new VisualElement();
+            var stub = new VisualElementConcealer(vis);
+            stub.HandleSettingChange(new HideCollectionEvent().WithValue(true).WithTarget(vis));
+            stub.SetVisibility(true);
+            Assert.True(vis.ClassListContains("hidden"));
```

Comment:
Nit: prefer `IsTrue/IsFalse` over `True/False` since it makes the test intent clearer (and matches other asserts like `AreEqual`).